eclair-core/src/main/scala/fr/acinq/eclair/db/Databases.scala

```diff
@@ -241,19 +244,22 @@ object Databases extends Logging {
 
       databases
     }
+  }
 
-    private def checkIfDatabaseUrlIsUnchanged(url: String, urlFile: File): Unit = {
-      def readString(path: Path): String = Files.readAllLines(path).get(0)
+  /** We raise this exception when the jdbc url changes, to prevent using a different server involuntarily. */
+  case class JdbcUrlChanged(before: String, after: String) extends RuntimeException(s"The database URL has changed since the last start. It was `$before`, now it's `$after`. If this was intended, make sure you have migrated your data, otherwise your channels will be force-closed and you may lose funds.")
```

Comment:
Very nit: maybe we can rename this to `DbUrlChanged` which is more generic?

Reply:
I don't think it's worth it since we still have references to "jdbc" or even "url" elsewhere.

Reply:
Fair enough